core/templates/domain/utilities/url-interpolation.service.ts

```diff
@@ -76,8 +81,9 @@ export class UrlInterpolationService {
   }
 
   /**
-   * Given a resource path relative to subfolder in /,
-   * returns complete resource path with cache slug and prefixed with url
+   * @param {string} prefix - A path to a subfolder in /.
```

Comment:
This should be `The url prefix` instead of `A path to a subfolder in/.`.

Reply:
Thanks